nano/node/unchecked_map.cpp

```diff
@@ -5,7 +5,8 @@
 #include <nano/lib/timer.hpp>
 #include <nano/node/unchecked_map.hpp>
 
-nano::unchecked_map::unchecked_map (nano::stats & stats, bool const & disable_delete) :
+nano::unchecked_map::unchecked_map (unsigned & max_unchecked_blocks, nano::stats & stats, bool const & disable_delete) :
```

Comment:
If you are going to use a reference then you should declare it as const since this function shouldn't be changing it. But I wouldn't use a reference for an int, just pass it by value.

Reply:
Fixed